packages/cxl-lumo-styles/scss/themes/vaadin-accordion-panel.scss

```diff
@@ -161,3 +161,48 @@
     padding-bottom: var(--lumo-space-m);
   }
 }
+
+:host([theme~="cxl-hub-sidebar"]),
+:host([theme~="cxl-dashboard-sidebar"]) {
```

Comment:
We can't apply multiple themes to panel and absolutely have to duplicate styles?

Reply:
you mean we should do something like this ?
`<vaadin-accordion-panel theme="cxl-dashboard-sidebar cxl-hub-sidebar">`

Reply:
Exactly.